events/religion_events/faith_conversion_events.txt

```diff
@@ -135,9 +135,9 @@ faith_conversion.0004 = {
 		# if = {
 			# limit = {
 				# OR = {
-					# culture_group = culture_group:arabic_group
-					# culture_group = culture_group:iranian_group
-					# culture = culture:turkish
+					# culture_group = culture_group:orc_group
+					# culture_group = culture_group:dwarf_group
+					# culture = culture_group:tauren_group
 				# }
 			# }
 			# change_government = clan_government
```

Comment:
# meants it's commented out. Thus the game doesn't read it. You have to uncomment it by removing # here.